addons/shaker/plugin.cfg

```diff
@@ -0,0 +1,7 @@
+[plugin]
+
+name="Shaker"
+description="Shake any properties of a node. You could use it as a camera shake effect."
+author="Lenny"
+version="1.0"
+script="plugin.gd"
```

Comment:
Is this one? https://github.com/Eneskp3441/Shaker . The format of `plugin.cfg` doesn't have a field for the plugin source, which doesn't help.

Reply:
It's this one: https://github.com/Rofle44-git/shaker-plugin-gd4